### ISSUE TITLE
Diff between build

### ISSUE THREAD

**Issue by thibaultduponchelle** [melezhik/sparky#32]
I would like to be able to compare builds, especially when it goes from OK to KO. 

It means a view where we can see these kind of thing, to be implemented as best effort :D
- the config/sparrowfile diffs if any
- the input diffs (commits if coming from scm, md5sum or similar if tarball) if any
- log diffs
- the diffs between docker images version (only the versions, not deeper)
- the diff between runner

**Comment by melezhik** [comment]
Yeah, it's doable. At least for `sparky.yaml` , `sparrowfile` right now ...
